lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,15 +1,7 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
 
 module SemanticPuppet
-  if defined?(GettextSetup)
-    GettextSetup.initialize(File.absolute_path('semantic_puppet/locales', File.dirname(__FILE__)))
-  end
```

Comment:
But, just asking for `defined?(GettextSetup)` does not create a hard dependency - does it? It just checks if that name is defined...